docs/components/console/manage-clusters/manage-api-clients.md

```diff
@@ -40,6 +40,14 @@ When the rate limit is triggered, the client will receive an HTTP 429 response.
 
 ## Create a client
 
+Currently, C8 SaaS is supporting the following Scopes:
+
+- Zeebe - Access to the Zeebe gRPC+REST API
+- Tasklist - Access to the Tasklist GraphQL API
```

Comment:
Tasklist GraphQL API is now deprecated as an FYI.

Reply:
ha, i stole these words from camunda academy ;)